thinc/util.py

```diff
@@ -437,6 +439,16 @@ def validate_fwd_input_output(
         sig_args["Y"] = (annot_y, ...)
         args["Y"] = (Y, lambda x: x)
     ArgModel = create_model("ArgModel", **sig_args)
+    ArgModel.update_forward_refs(
+        Floats2d=Floats2d,
+        Floats1d=Floats1d,
+        Floats3d=Floats3d,
+        Floats4d=Floats4d,
+        Ints1d=Ints1d,
+        Ints2d=Ints2d,
+        Ints3d=Ints3d,
+        Ints4d=Ints4d
```

Comment:
I wonder if we should just pass in all the `**types` here? Because in theory, there are more types that the forward functions could use. And using the `globals()` is more vague, and makes it less obvious that the imports in `util.py` matter.

Reply:
Okay, it can't be `**types` because that's a module, but you get the idea 😅

Reply:
I see. I guess it's fine now? :kissing:

Reply:
Yeah, I think the solution I came up with works and does what we want 🎉 Thanks again!